frontend/src/app/components/files.service.ts

```diff
@@ -55,6 +57,21 @@ export class FilesService {
   }
 
   public deleteFile(id: number) {
-    console.log("Waiting for implementation :(.")
+    if (this.files.has(id)) {
+      this.http.delete<FileDeleteRespons>(`http://localhost:8088/api/v1/delete/${id}`).subscribe(
+        (response: FileDeleteRespons) => {
+          if (response.sucess) {
```

Comment:
In my opinion, completion status of the delete request should be in response code (ex. 200, 404 etc.), not in response body.